pkg/sdkclient/sinker/client.go

```diff
@@ -38,23 +36,13 @@ type client struct {
 
 var _ Client = (*client)(nil)
 
-// New creates a new client object.
+// New creates a new client object. Sinker doesn't require server info to start ATM.
```

Comment:
Is it true? https://github.com/numaproj/numaflow-go/blob/main/pkg/sinker/server.go#L39

Reply:
The source and sinker clients don't require serverinfo to start.. fixed the godoc